Cargo.toml

```diff
@@ -61,6 +60,27 @@ fakeit = "1.1"
 rand = "0.8"
 pretty_assertions = "1.4.0"
 
+[features]
+default = ["crossterm"]
+serde = ["dep:serde", "bitflags/serde"]
```

Comment:
Since it includes more than just the `serde` crate, I'd document the `serde` feature here. I think it's more natural when reading the source code.

Reply:
Makes sense, 8bc8d5c6b925554ccd5e24d4b938f5b7d5a4d164